stix_shifter_modules/aws_athena/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,522 @@
+      "service_action_awsapicallaction_remoteipdetails_ipaddressv4": [
+        {
+          "key": "ipv4-addr.value",
+          "object": "api_remote_ip"
+        },
+        {
+        "key": "x-ibm-finding.dst_ip_ref",
+        "object": "ibm_finding",
+        "references": "api_remote_ip"
+        },
+        {
+        "key": "x-ibm-finding.dst_geolocation",
+        "object": "ibm_finding",
+        "ds_key": "service_action_awsapicallaction_remoteipdetails_country_countryname",
+        "cybox": false
+        },
+        {
+          "key": "ipv4-addr.x_aws_remote_city_name",
+          "object": "api_remote_ip",
+          "ds_key": "service_action_awsapicallaction_remoteipdetails_city_cityname",
+          "cybox": false
+        },
+        {
+          "key": "ipv4-addr.x_aws_remote_country_name",
+          "object": "api_remote_ip",
+          "ds_key": "service_action_awsapicallaction_remoteipdetails_country_countryname",
+          "cybox": false
+        }
+      ],
+      "resource_accesskeydetails_principalid": {
+          "key": "user-account.user_id",
+          "object": "api_user"
+      },
+      "resource_accesskeydetails_username": {
+        "key": "user-account.account_login",
+        "object": "api_user"
+      },
+      "resource_accesskeydetails_accesskeyid": {
+        "key": "x-aws-api.access_key_id",
+        "object": "api_details"
+      },
+      "service_action_awsapicallaction_api": {
+        "key": "x-aws-api.api",
+        "object": "api_details"
+      },
+      "service_action_awsapicallaction_servicename": {
+        "key": "x-aws-api.service_name",
+        "object": "api_details"
+      },
+    "dnsrequest_resource_instancedetails_networkinterfaces_0_privateipaddress":[
+      {
+        "key": "ipv4-addr.value",
+        "object": "dns_private_ip1"
+      },
+      {
+        "key": "domain-name.resolves_to_refs",
+        "object": "private_dns_name",
+        "references": ["dns_private_ip1"]
+      },
+      {
+        "key": "x-ibm-finding.src_ip_ref",
+        "object": "ibm_finding",
+        "references": "dns_private_ip1"
+      },
+      {
+        "key": "ipv4-addr.x_aws_interface_id",
+        "object": "dns_private_ip1",
+        "ds_key": "resource_instancedetails_networkinterfaces_0_networkinterfaceid",
+        "cybox": false
+      },
+      {
+        "key": "ipv4-addr.x_aws_ip_type",
+        "object": "dns_private_ip1",
+        "value": "private",
+        "cybox": false
+      }
+    ],
+  "service_action_dnsrequestaction_domain": [
+        {
+          "key": "domain-name.value",
+          "object": "dns_domain_name"
+        }
+      ],
+  "accountid": {
+      "key": "x-aws-details.account_id",
+      "object": "aws_details"
+    },
+  "region": {
+    "key": "x-aws-details.region",
+    "object": "aws_details"
+  },
+  "severity": {
+      "key": "x-ibm-finding.severity",
+      "object": "ibm_finding"
+    },
+  "id":{
+    "key": "x-ibm-finding.finding_id",
```

Comment:
We never specified that the finding object should have an id so I think this can be removed.

Reply:
@delliott90 The finding id is now removed from the to_stix mapping and the code is updated in PR branch